llvm/test/CodeGen/SystemZ/builtin-setjmp-spills-double-03.ll

```diff
@@ -0,0 +1,216 @@
+; Simulate register pressure  around setjmp call for double precision 
+; arguments and return sum of 20 vaiables. It also prints the variables.
```

Comment:
As this is not an executable test, I don't think the printf code-gen tests anything relevant.   The above register pressure test should be sufficient.